app/assets/javascripts/script.js

```diff
@@ -0,0 +1,11 @@
+
+$(document).on('ready turbolinks:load', () => {
+	scrollerSignUp();
+})
+function scrollerSignUp(){
+	$('#productscrolldown').on('click', function(){
+		// debugger
+		$('#scroller').removeClass('d-none');
+
+	})
+}
```

Comment:
Remove commented code and add newline at the end of file. Also correct the indentation issues